ballerina-command/src/main/java/org/ballerinalang/command/cmd/UpdateToolCommand.java

```diff
@@ -88,10 +88,10 @@ public void setParentCmdParser(CommandLine parentCmdParser) {
 
     private static void updateCommands(PrintStream printStream) {
         String version = ToolUtil.getCurrentToolsVersion();
-        printStream.println("Fetching latest version from remote server...");
+        printStream.println("Fetching the latest version from the remote server...");
         String latestVersion = ToolUtil.getLatestToolVersion();
         if (latestVersion.equals(version)) {
-            printStream.println("You are already in latest command version: " + latestVersion);
+            printStream.println("The command is already in the latest version: " + latestVersion);
```

Comment:
Don't we have a proper term to refer to the tool?

Reply:
@keizer619 @Kishanthan

Reply:
@pubudu91 I think It should be referred as "command". With installers what user get is the ballerina command which has all the sub commands handled by a ballerina distribution. Ballerina distribution itself could be managed with "dist" subcommand.